examples/restrict-by-media-type/restrict_by_media_type.bal

```diff
@@ -1,7 +1,7 @@
 import ballerina/http;
 import ballerina/log;
 
-// Consumes and produces annotations that contain MIME types as an array of strings.
+// The `consumes` and `produces` annotations that contain MIME types as an array of strings.
```

Comment:
This description seems out of place. Shall we add a modified version of this to the comment below (the one for the resource config)?